src/templates/post.js

```diff
@@ -82,21 +83,59 @@ const BlogPost = ({ data }) => {
     };
   }, []);
 
+
   const handleScroll = () => {
     const moreFromXlSize = refMoreFrom?.current?.clientHeight || 0;
-     const isInbottom = Math.ceil(window.innerHeight + window.scrollY + moreFromXlSize + 200) >= document.documentElement.scrollHeight;
-     if (isInbottom) {
-       setDisappearSocial(true);
-     } else {
-       if (!disappearSocial) {
-         setDisappearSocial(false);
-       }
-     };
-   }; 
-  
+    const isInbottom = Math.ceil(window.innerHeight + window.scrollY + moreFromXlSize + 200) >= document.documentElement.scrollHeight;
+    const isInTop = document.documentElement.scrollTop < 800;
+    
+    if (isInbottom) {
+      setDisappearSocial(true);
+    } else {
+      if (!disappearSocial) {
+        setDisappearSocial(false);
+      }
+    };
+    if (isInTop || isInbottom) {
+      setDisappearIndex(true);
+    } else {
+      if (!disappearIndex) {
+        setDisappearIndex(false);
+      }
```

Comment:
this is unnecessary, when disappearIndex is false this if is true and you set disappearIndex as false

Reply:
I don't have the same opinion as you

Reply:
I'm a bit lost about the method, so, I'm going to check the code and tell you my resolution about that line

Reply:
My mistake, we already discussed this and I forgot. Is ok